packages/taxdat/R/stan_input_helpers.R

```diff
@@ -1610,7 +1612,9 @@ impute_adm0_obs_single <- function(sf_cases_resized,
     dplyr::filter(ref_TL == m_TL,
                   ref_TR == m_TR) 
   
-  # All national level data (censored)
+  # All national level data (censored) ## ECL: So here, we presume these are all censored because
+  # this function only gets called within impute_adm0_obs? It's a bit sloppy... Should we add the
```

Comment:
It seems weird to refer to this as censored adm0 obs in the comment when that is dependent on code in the wrapper function impute_adm0_obs

Reply:
This is a copy/paste error. This is all data.